tests/modules/nf-core/freebayes/test.yml

```diff
@@ -3,42 +3,45 @@
   tags:
     - freebayes
   files:
-    - path: output/freebayes/test.vcf.gz
-      md5sum: 4978a2f69d7419db9e5a740cba925d19
+    - path: output/freebayes/test.vcf.gz # No md5sum because the file includes a timestamp
+      contains:
+        - "MT192765.1\t10214\t.\tATTTAC\tATTAC\t29.8242"
     - path: output/freebayes/versions.yml
 
 - name: freebayes test_freebayes_bed
   command: nextflow run ./tests/modules/nf-core/freebayes -entry test_freebayes_bed -c ./tests/config/nextflow.config
   tags:
     - freebayes
   files:
-    - path: output/freebayes/test.vcf.gz
-      md5sum: dfb4e3539ebb058ca966d6185e139063
+    - path: output/freebayes/test.vcf.gz # No md5sum because the file includes a timestamp
     - path: output/freebayes/versions.yml
 
 - name: freebayes test_freebayes_cram
   command: nextflow run ./tests/modules/nf-core/freebayes -entry test_freebayes_cram -c ./tests/config/nextflow.config
   tags:
     - freebayes
   files:
-    - path: output/freebayes/test.vcf.gz
-      md5sum: 178263bc22b281fa3a664835b3a8e7c8
```

Comment:
why did the md5sum work so far 😱

Reply:
It seems to work as long as the checksums are generated on the author's compute environment and tested on GitHub on the same day.

Check the last PR that updated those, #2493 (by you 😀 ):

1. You made an initial change on Nov 10: https://github.com/nf-core/modules/pull/2493/commits/83da8f147c9704ae6a0fe132d992e5c493d2c3c8 and I'm sure it all worked at the time
2. There were other changes to make and the pull-request went on to Nov 11 too. Then, you had to update the same checksums again: https://github.com/nf-core/modules/pull/2493/commits/f3243c8b8a38a0d042ed0de9ede2fbfdb3fa5080

Reply:
makes sense, so just the date no time :D